app.js

```diff
@@ -21,10 +21,10 @@ db.on("error", err => console.log(err.message + " is Mongod not running?"));
 db.on("connected", () => console.log("mongo connected: ", mongoURI));
 db.on("disconnected", () => console.log("mongo disconnected"));
 
-// Goal.create(goalSeed, (err, data) => {
-//   if (err) console.log(err.message);
-//   console.log("added provided goal data");
-// });
+Goal.create(goalSeed, (err, data) => {
+  if (err) console.log(err.message);
+  console.log("added provided goal data");
```

Comment:
This only needs to run once. I would leave it commented out since our schema only has one goal, it will keep adding the same goal every time you run it.

Reply:
That makes sense, nice catch!